aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/handlers/container/common/ContainerInfo.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.core.handlers.container.common;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Represents an aggregate of Tab or Accordion instance and a list of fields designed to be rendered within this container element.
+ * Used to compose a sorted "container element registry" for a component class
+ */
+public class ContainerInfo {
+    private String title;
+    private Map<String, Object> attributes = new HashMap<>();
```

Comment:
Can we put initialization in the constructor as the initialization of the field's map?

Reply:
Fixed